cumulusci/tasks/github/release.py

```diff
@@ -68,6 +71,8 @@ def _run_task(self):
         message = self.options.get("message", "Release of version {}".format(version))
         if self.options.get("version_id"):
             message += f"\n\nversion_id: {self.options['version_id']}"
+        if self.options.get("package_type"):
+            message += f"\npackage_type: {self.options['package_type']}"
```

Comment:
The other message concatenations use two newlines as a prefix to each data element. Let's stick with that to avoid any future confusion.